core/serialization.go

```diff
@@ -362,6 +363,19 @@ func (h *BlobHeader) Serialize() ([]byte, error) {
 
 func (h *BlobHeader) Deserialize(data []byte) (*BlobHeader, error) {
 	err := decode(data, h)
+
+	if !(*bn254.G1Affine)(h.BlobCommitments.Commitment).IsInSubGroup() {
+		return nil, fmt.Errorf("in BlobHeader Commitment is in the subgroup")
```

Comment:
is `not` in subgroup

Reply:
we want to reject points that is not part of subgroup, which has undefined operation for (G1, G2)point operation

Reply:
typo as well

Reply:
I see, thanks!